etcdserver/server.go

```diff
@@ -629,10 +636,12 @@ func (s *EtcdServer) applyAll(ep *etcdProgress, apply *apply) {
 		plog.Warningf("avoid queries with large range/delete range!")
 	}
 	proposalsApplied.Set(float64(ep.appliedi))
+	s.applyWait.Trigger(ep.appliedi)
```

Comment:
should this be after `raftDone`? otherwise it could ack commits that haven't been persisted?